pkg/modulewriter/tfwriter.go

```diff
@@ -119,10 +111,17 @@ func writeOutputs(
 		}
 	}
 
-	// Write file
 	hclBytes := hclFile.Bytes()
+	if len(hclBytes) == 0 {
```

Comment:
Let's rely on number of outputs, not byte-size of tokenized HCL.
And return empty list, not nil.